src/pages/Categories.js

```diff
@@ -0,0 +1,11 @@
+function Catagories() {
+  return (
+    <>
+      <div className="h-4/5 flex items-center justify-center">
+        <button type="button" className="bg-sky-400 text-white rounded py-2 px-16 mt-2px">Update Progress</button>
```

Comment:
[OPTIONAL]
- Please change the text from `Update Progress` to `Check Status` as required.